src/main/resources/liquibase/crop_changelog/18_1_0.xml

```diff
@@ -214,4 +214,244 @@
         </sql>
 	</changeSet>
 
+	<changeSet author="lkovacic" id="v18.1.0-13">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="audit_cfg"/>
+			</not>
+		</preConditions>
+		<comment>
+            Add 'audit_cfg' table
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+			create table audit_cfg (
+				id          int auto_increment
+					primary key,
+				entity_name varchar(255)         not null,
+				is_audited  tinyint(1) default 1 not null,
+				constraint audit_cfg_entity_name_uindex
+					unique (entity_name)
+			);
+        </sql>
+	</changeSet>
+
+	<changeSet author="lkovacic" id="v18.1.0-14">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">SELECT count(1) FROM audit_cfg;</sqlCheck>
+		</preConditions>
+		<comment>
+            Add data to 'audit_cfg' table
+        </comment>
+        <sql dbms="mysql" splitStatements="true">
+			INSERT INTO audit_cfg (entity_name, is_audited)
+			VALUES ('GERMPLSM', 1),
+				('ATRIBUTS', 1),
+				('BIBREFS', 1),
+				('NAMES', 1),
+				('EXTERNAL_REFERENCE', 1),
```

Comment:
I think we can just rename it to GERMPLASM_EXTERNAL_REFERENCE now so we dont affect this row when renaming the table @lik-83